lib/cocoapods/sandbox/path_list.rb

```diff
@@ -52,9 +52,18 @@ def read_file_system
         end
         escaped_root = escape_path_for_glob(root)
 
-        absolute_paths = Pathname.glob(escaped_root + '**/*', File::FNM_DOTMATCH).lazy
```

Comment:
also `.lazy` part no longer needed?

Reply:
I think the lazy broke some of the next lines, but I don't know enough about Ruby to know why. I initially put it back but then removed it again

Reply:
Lets make sure we understand the intention here before we merge this.

http://ruby-doc.org/core-2.0.0/Enumerator/Lazy.html

Reply:
And http://blog.honeybadger.io/using-lazy-enumerators-to-work-with-large-files-in-ruby/

so I think it was for performance reasons which might be important especially for very large projects like ours.

Reply:
Yup, totally agree. Mentioned in the other comment that I put it back and everything seems okay, I had forked robert's repo which had it removed